packages/react-app/src/pages/manage.scss

```diff
@@ -0,0 +1,96 @@
+@import '../foundation.scss';
+
+#create-new-accordion {
+  width: 175px;
+  border-radius: 10px;
+  background-color: transparent;
+}
+
+#create-new-accordion-summary {
+  padding: 0;
+  margin: 0;
+  min-height: 40px;
+}
```

Comment:
Since this is a SCSS file, you could also do things like this to keep the CSS organized:
```
#create-new-accordion {
  width: 175px;
  border-radius: 10px;
  background-color: transparent;

  &-summary {
    padding: 0;
    margin: 0;
    min-height: 40px;
  }

  .accordion-details {
    padding: 0;
    margin-top: 9px;
  }
}
```